app/assets/stylesheets/common/base/directory.scss

```diff
@@ -29,6 +29,21 @@
     .filter-name {
       height: 1em;
     }
+    .select-kit.multi-select {
```

Comment:
can you try to use flexbox on parent instead of all these manual fixes? you are not supposed to change internat alignment of select-kit too, why is it needed here ? is some other css interfering?

Reply:
Yes, the internal CSS of the select-kit doesn't allow to change the height.